app/helpers/administrate/application_helper.rb

```diff
@@ -32,18 +32,12 @@ def resource_index_route_key(resource_name)
       ActiveModel::Naming.route_key(class_from_resource(resource_name))
     end
 
-    def sanitized_order_params(page, current_field_name)
-      collection_names = page.association_includes + [current_field_name]
-      association_params = collection_names.map do |assoc_name|
-        { assoc_name => %i[order direction page per_page] }
-      end
-      params.permit(:search, :id, :page, :per_page, association_params)
+    def sanitized_order_params
+      params.permit!
     end
 
     def clear_search_params
-      params.except(:search, :page).permit(
-        :per_page, resource_name => %i[order direction]
-      )
+      params.except(:search, :page).permit!
```

Comment:
Similarly to my other comment:

```suggestion
      params.except(:search, :page).to_unsafe_h
```